clients/drcachesim/tracer/func_trace.cpp

```diff
@@ -480,7 +502,10 @@ func_trace_init(func_trace_append_entry_vec_t append_entry_vec_,
             goto failed;
         }
     }
-    if (!drwrap_init()) {
+    /* For multi-instrumentation cases with drbbdup, we need the drwrap inverted
+     * control mode where we invoke its instrumentation handlers.
+     */
+    if (!drwrap_set_global_flags(DRWRAP_INVERT_CONTROL) || !drwrap_init()) {
```

Comment:
I'm looking at the docs for `DRWRAP_INVERT_CONTROL`, and it makes sense use it here. 

Perhaps unrelated, but can its docs be clearer when it says "As this is a global change in how drwrap operates, be careful that its use does not conflict with drwrap uses in any libraries or joint clients. " Does this mean that all libraries must use DRWRAP_INVERT_CONTROL if only one must have it on?

Reply:
Thinking about it more: If multiple parties use DRWRAP_INVERT_CONTROL then the drwrap callbacks will be called too many times and they'll do duplicate work: so it feels kind of like the opposite of that quote  Adding some kind of do-once lock could maybe help -- but what if one user wants drwrap in one place and the other in another that are not compatible (like different drbbdup cases)?  Not sure it's easy to solve in an automated fashion.

If only one use wants inversion, and several others are fine with whatever (and fine with no wrapping in the inversion user's drbbdup cases), it should all work out: so it's not quite like the quote.

I think the biggest problem is if a user wants inversion and someone else wants drwrap replacement: then we have a problem since that's not supported (but: integrating drwrap into drmgr and the app2app phase could solve that -- I filed #5400 so we can point at something and record things there).

I augmented the docs for this flag.